includes/core-changes/aspnetcore/3.1/http-cookie-samesite-authn-impacts.md

```diff
@@ -14,28 +14,28 @@ For discussion on this issue, see [aspnet/AspNetCore#14996](https://github.com/a
 
 #### New behavior
 
-Google is pushing a new draft standard that isn't backwards compatible. The standard changes the default mode to `Lax` and adds a new entry `None` to opt out. `Lax` suffices for most app cookies; however, it breaks cross-site scenarios like OpenID Connect and WS-Federation login. Most OAuth logins aren't affected due to differences in how the request flows. The new `None` parameter causes compatibility problems with clients that implemented the prior draft standard (for example, iOS 12). Chrome plans to go live with their changes in Chrome 80 in February 2020.
+Google proposed a new draft standard that isn't backwards compatible. The standard changes the default mode to `Lax` and adds a new entry `None` to opt out. `Lax` suffices for most app cookies; however, it breaks cross-site scenarios like OpenID Connect and WS-Federation login. Most OAuth logins aren't affected because of differences in how the request flows. The new `None` parameter causes compatibility problems with clients that implemented the prior draft standard (for example, iOS 12). Chrome 80 will include the changes. See [SameSite Updates](https://www.chromium.org/updates/same-site) for the Chrome product launch timeline.
 
 ASP.NET Core 3.1 has been updated to implement the new `SameSite` behavior. The update redefines the behavior of `SameSiteMode.None` to emit `SameSite=None` and adds a new value `SameSiteMode.Unspecified` to omit the `SameSite` attribute. All cookie APIs now default to `Unspecified`, though some components that use cookies set values more specific to their scenarios such as the OpenID Connect correlation and nonce cookies.
```

Comment:
```suggestion
ASP.NET Core 3.1 has been updated to implement the new `SameSite` behavior. The update redefines the behavior of <xref:Microsoft.AspNetCore.Http.SameSiteMode.None?displayProperty=nameWithType> to emit `SameSite=None` and adds a new value `SameSiteMode.Unspecified` to omit the `SameSite` attribute. All cookie APIs now default to `Unspecified`, though some components that use cookies set values more specific to their scenarios such as the OpenID Connect correlation and nonce cookies.
```

Reply:
As discussed on Teams, let's pass on this suggestion. The `SameSiteMode` enum lives in 2 different assemblies: `Microsoft.Net.Http.Headers.SameSiteMode` and `Microsoft.AspNetCore.Http.SameSiteMode`.